### ISSUE TITLE
False positive max and min in scss/no-global-function-names

### ISSUE THREAD

**Issue by jkevingutierrez** [stylelint-scss/stylelint-scss#560]
Input:

```
div {
  min-width: max(30vw, 300px);
}
```

Require:

```
div {
  min-width: math.max(30vw, 300px);
}
```
Expected (no errors):

```
div {
  min-width: max(30vw, 300px);
}
```

**Comment by sixclones** [comment]
Also happening on `filter: invert(1);`.
